tests/services/test_strategy_marketplace_integration.py

```
@@ -0,0 +1,209 @@
+
+            await submission_service.review_submission(
+                submission_id=submission.id,
+                reviewer=reviewer_stub,
+                action="approve",
+                comment="Looks good",
+                db=session,
+            )
+
+        original_live_performance = marketplace_service._get_live_performance
+
+        async def _fake_live_performance(strategy_id: str, session=None):  # type: ignore[override]
```

Ruff (0.13.1)

Unused function argument: `strategy_id`

(ARG001)